MangoAPI.Application/Interfaces/IBlobService.cs

```diff
@@ -5,7 +5,7 @@ namespace MangoAPI.Application.Interfaces;
 
 public interface IBlobService
 {
-    Task<string> GetBlobAsync(string fileName);
+    string GetBlobAsync(string fileName);
```

Comment:
The name `GetBlobAsync` suggests that it is async method, but its not, so should be renamed to `GetBlob`